demisto_sdk/commands/common/hook_validations/release_notes.py

```diff
@@ -50,7 +53,7 @@ def are_release_notes_complete(self):
                 elif self.pack_name + '/' in file:
                     # Refer image and description file paths to the corresponding yml files
                     file = UpdateRN.check_for_release_notes_valid_file_path(file)
-                    update_rn_util = UpdateRN(pack_path=self.release_notes_file_path, modified_files_in_pack=set(),
+                    update_rn_util = UpdateRN(pack_path=self.pack_path, modified_files_in_pack=set(),
```

Comment:
can we add test?

Reply:
we can but this fixes just an incorrect print that happened not an incorrect behavior so the test seems kinda redundant and will probably not look good. The changed lines are covered in other existing tests.